content/docs/user-guide/structure-of-remote-cache-by-example.md

```diff
@@ -0,0 +1,251 @@
+
+## Pushing the DVC-tracked files to the remote cache
+
+Now come's the interesting part!
+
+> To fully appreciate how remote caches work, it's useful
+> to understand the concept of _indirection_. Indirection refers
+> to the naming of things using a reference.  
+> For example, my name is "Mike", which is one indirect way of referring to the person that is me. 
+> Another way of referring to me could "Employee #25531" used by my company. 
+> Two indirect references to the same one person.
+
+Back to the project.
+Let's now push our four data files to the remote cache:
+
+```dvc
+$ dvc push -r s3cache
+1 file pushed
+```
+
+Notice the output of this command: `1 file pushed`.
+If you were following through the previous section,
+you should know that _four_ data files were created.
+You can double check this if you like:
+
+```dvc
+% ls
+data1		data2		data3		data4		
+data1.dvc	data2.dvc	data3.dvc	data4.dvc	
+```
+
+So why was only _one_ data file pushed?
+
+If you have the `aws` command line tool set up
+(or whatever your equivalent is, you could also browse your cache via the web),
+you could further check the contents of the remote cache.
+Here's what the S3 cache looked like on my system:
+
+```dvc
+$ aws s3 ls s3://path/to/your/cache/
+                           PRE 61/
+2022-02-20 13:16:49          0
+```
+
+One folder `61/` has been created, and if you look inside this, the folder contains only a single file with an obfuscated (hexademical) name.
+
+So what's going on?
+
+The answer, of course, is that the four data files were identical and so DVC concluded that only one copy of the files really needed to be uploaded.
+The name `61/` is an internal name for the file used to implement an indirection.
+So the file `data1` on my local computer references `61/` on the remote cache, the file `data2` on my local computer also references `61/`,
+and so on.
+
+DVC determines which files are identical by comparing their MD5 hashes, and you can confirm this by looking at the `.dvc` files for the four data files yourself: 
+
+```dvc
+$ cat *.dvc
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data1
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data2
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data3
+outs:
+- md5: 6137cde4893c59f76f005a8123d8e8e6
+  size: 5
+  path: data4
+```
+
+Graphically, the project folder and the remote cache have this structure:
+
+![Data in project and remote cache](../../../static/img/project_remote_1.png)
+
+## Adding new data and how the remote cache changes
+
+Let's add two more data files to our repository, but this time make them  different to the first four data files. We should also make sure DVC tracks the new files and add the corresponding `.dvc` files with git: 
+
+```dvc
+$ echo data3636363 > data5
+$ echo blahblahblah > data6
```

Comment:
Minor: I think it looks better to use more meaningful samples even if the data contents are irrelevant for the story.